### PR TITLE
Added Admin Center Module

### PR DESCRIPTION
**Pull request by bamajeed**
@sebassem I have created the Admin Center Module, please review

### CODE REVIEW

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 1]
```diff
@@ -659,10 +659,48 @@ if ($roleDefinitionIds.Count -gt 0)
 ### Module 10: Manage your Azure Arc-enabled servers using Admin Center (Preview)
 
 #### Module overview
+In this module you will learn how to use the Windows Admin Center in the Azure portal to manage the Windows operating system of your Arc-enabled servers, known as hybrid machines. You can securely manage hybrid machines from anywhere without needing a VPN, public IP address, or other inbound connectivity to your machine.
 
-#### Task 1
+#### Task 1: Pre-requisites
 
-#### Task 2
+Pre-requisite 1: Check the status of the resource provider _Microsoft.HybridConnectivity_ and register it if not registered.
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:666]: No need for this step as the automation already registers the resource providers for the user

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 2]
```diff
@@ -659,10 +659,48 @@ if ($roleDefinitionIds.Count -gt 0)
...
+
+- If you get the following message after the installation is complete then you need to go back to the previous step and set up the permissions as explained in Pre-requisite 2.
+
+![Screenshot permissions missing for Admin Centre](./Admin_Centre_install_message_1.png)
+#### Task 3: Connect and explore Windows Admin Center (preview)
+
+- Once the installation is complete then you can connect to the Windows Admin Center 
+![Screenshot connecting to Admin Center](./Admin_Center_Connect.png)
+
+- Start exploring the capabilities offered by the Windows Admin Center to manage your Arc-enabled Windows machine.
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:702]: Let's do one specific action, like adding a role, running a script, creating a local group/user,....etc What do you think ?

**Review by sebassem — CHANGES_REQUESTED** [review]
Looks great, just 2 minor comments

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md (hunk 2 above)
Comment (bamajeed) [reply docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:702]: To keep it short and easy I will create a group called "Partner Users", I will create a user with user name "Janis" and add the user to the group. Anything else might take time and the lab could already be long. What do you think?

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md (hunk 1 above)
Comment (bamajeed) [reply docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:666]: will do

**Comment by bamajeed** [comment]
I have added the users and groups in Admin Center

**Pull request merged**